src/sentry/static/sentry/app/components/group/releaseStats.tsx

```diff
@@ -75,8 +80,17 @@ const GroupReleaseStats = ({
             secondary
             title={
               <React.Fragment>
-                <span>{t('Last seen')}</span>
-                {environments.length > 0 && <small>({environmentLabel})</small>}
+                <span>
```

Comment:
Is the span needed here? If it is, then we don't need `Fragment`

Reply:
the span is needed here, so I removed `Fragment`